fairseq_utility.py

```diff
@@ -0,0 +1,78 @@
+import shlex
+import shutil
```

Comment:
Where have we used `shutil` in this file? If not let's remove this line

Reply:
shutil is used as a pre requisite as many files are generated after firing a commands related to fairseq. This library makes files copy, create, etc operations easier and faster.

Reply:
shutil is used as a pre requisite as many files are generated after firing a commands related to fairseq. This library makes files copy, create, etc operations easier and faster.